nautobot/core/api/fields.py

```diff
@@ -113,14 +113,15 @@ def __init__(self, *args, read_only=True, **kwargs):  # pylint: disable=useless-
         """Default read_only to True as this should never be a writable field."""
         super().__init__(*args, read_only=read_only, **kwargs)
 
-    def to_representation(self, _value):
-        """
-        Get the content-type of this serializer's model.
+    def get_attribute(self, _instance):
+        """Get the content-type of this serializer's model."""
+        model = self.parent.Meta.model
+        return model._meta.label_lower
 
-        Implemented this way because `_value` may be None when generating the schema.
-        """
+    def to_representation(self, _value):
+        """Get the content-type of this serializer's model."""
         model = self.parent.Meta.model
-        return f"{model._meta.app_label}.{model._meta.model_name}"
+        return model._meta.label_lower
```

Comment:
Took me a minute to parse. These seem identical. I'm sure we're overloading the bases but should we not have one call the other or make a common function for them?

Reply:
I mean it's two lines of code :-). I'm honestly not sure why I needed to add a `get_attribute()` method to this class when it wasn't needed before, but object-type fields weren't being included properly until I added it. My best guess is that since it's not an actual model field, it wasn't being included by default in `__all__`?

Reply:
I wonder if that is the cause of some of my weird sort behavior in #3708